recipes/install_ui.rb

```diff
@@ -51,3 +53,85 @@
   key_file node['abiquo']['certificate']['key_file']
   ca_file node['abiquo']['certificate']['ca_file']
 end
+
+include_recipe 'haproxy-ng::install'
+include_recipe 'haproxy-ng::service'
+
+if node['abiquo']['haproxy']['use_default_path']
+  first_path = node['abiquo']['haproxy']['ws_paths'].first
```

Comment:
[minor] Raise an error if the default path flag is set but no mappings are configured.